cfgov/unprocessed/css/molecules/related-posts.less

```diff
@@ -92,13 +99,29 @@
         }
     }
 
+    .list_item .date {
+        margin-bottom:0;
+        margin-top: unit(10px / @base-font-size-px, em);
+    }
     .respond-to-range(@bp-sm-min, @bp-sm-max, {
         @list_half-width();
     });
 }
 
-.m-related-posts {
-    padding-bottom: unit(@grid_gutter-width * 2 / @base-font-size-px, em);
+.m-related-posts_list-container {
+    margin-bottom: @margin_half__em * 2;
+
+    &:last-child {
```

Comment:
Alternatively, use `&:not( :last-child ) { margin-bottom: @margin_half__em * 2; }`. Both have the same level of support.